plugins/modules/notification/slack.py

```diff
@@ -234,7 +235,7 @@
 - name: Edit message
   community.general.slack:
     token: thetoken/generatedby/slack
-    channel: "{{ slack_response.channel }}"
+    channel: "{{ slack_response.channel }}" #doesn't accept channel name. Must be `channel_id` stored in `slack_response` from the previous task.
```

Comment:
This line is now way too long.
```suggestion
    # The 'channel' option does not accept the channel name. It must use the 'channel_id',
    # which can be retrieved for example from 'slack_response' from the previous task.
    channel: "{{ slack_response.channel }}"
```